compiler/rustc_mir_build/src/build/scope.rs

```diff
@@ -579,13 +580,13 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
             );
             let current_root = tcx.maybe_lint_level_root_bounded(current_hir_id, self.hir_id);
```

Comment:
Those 2 calls should go into `maybe_new_source_scope`.

Reply:
Ah yes I was just fixing this when you left his comment :-)